src/types.ts

```diff
@@ -1236,6 +1236,15 @@ export interface LiveEventMessage {
   tags: SyncTag[]
 }
 
+interface QueryReturn {}
+
+export interface SanityQueries extends QueryReturn {}
+
+export type ClientReturn<
+  GroqString extends string,
+  Fallback = Any,
+> = GroqString extends keyof SanityQueries ? SanityQueries[GroqString] : Fallback
```

Comment:
@sgulseth this type is slightly different from the type from your tsplayground we discussed on Slack: it is missing the `GeneratedOperations` argument.

It works like this without it and could not find a reason to include it, but I might be overlooking something. Let me know if you want me to include the `GeneratedOperations` argument.